diff --git a/node_modules/@sveltejs/adapter-auto/adapters.js b/node_modules/@sveltejs/adapter-auto/adapters.js
index e018e74..182f941 100644
--- a/node_modules/@sveltejs/adapter-auto/adapters.js
+++ b/node_modules/@sveltejs/adapter-auto/adapters.js
@@ -13,5 +13,10 @@ export const adapters = [
 		name: 'Netlify',
 		test: () => !!process.env.NETLIFY,
 		module: '@sveltejs/adapter-netlify'
+	},
+	{
+		name: 'Azure SWA',
+		test: () => process.env.GITHUB_ACTION_REPOSITORY === 'Azure/static-web-apps-deploy',
+		module: 'svelte-adapter-azure-swa'
 	}
 ];
